API.yaml

```diff
@@ -154,6 +154,91 @@ paths:
           description: id not found
       security:
         - token: []
+  /events/{id}/photos:
+    get:
+      tags:
+        - Event
+      summary: Get list of photo of event
+      description: Returns a list of photo paths
+      operationId: getPhoto
+      parameters:
+        - name: id
+          in: path
+          description: ID of event to return
+          required: true
+          schema:
+            type: integer
+            format: int64
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+                example: ['path_to_photo1', 'path_to_photo2', 'path_to_photo3']
+        '400':
+          description: Invalid ID supplied
+        '404':
+          description: Event not found
+    delete:
+      tags:
+        - Event
+      summary: Cancel event
+      operationId: deletePhoto
+      parameters:
+        - name: id
+          in: path
+          description: id of Event
+          required: true
+          schema:
+            type: string
+        - name: path
+          in: header
+          description: path of photo
+          required: true
+          schema:
+            type: string
```

Comment:
Why everything is always in header?

Reply:
It's public path, so imo it can be passed by header to avoid creating new obj.